src/components/add-note-form/AddNoteForm.jsx

```diff
@@ -57,11 +57,17 @@ export const AddNoteForm = ({ name }) => {
   const disabledClasses = createClassNames(styles.disabled, 'disabled');
   const btnSubmitClasses = createClassNames(btnClasses, 'btn-primary', { [disabledClasses]: formState.amount === '' || formState.description === '' });
   const cancelBtnClasses = createClassNames(btnClasses, 'btn-danger');
+  const inputClasses = createClassNames(styles['form-control'], 'form-control');
```

Comment:
rename import to 'cn', also you need naming classes with camelCase natation to prevent using ['form-control'],
in your scss file 
.formControl {...}
something like that >>> cn(styles.formControl, 'form-control')